plugin/mouse/mouse.py

```diff
@@ -38,8 +37,7 @@
 class Actions:
     def zoom_close():
         """Closes an in-progress zoom. Talon will move the cursor position but not click."""
-        if eye_zoom_mouse.zoom_mouse.state == eye_zoom_mouse.STATE_OVERLAY:
-            actions.tracking.zoom_cancel()
+        actions.user.deprecate_action("2024-12-26", "user.zoom_close")
```

Comment:
Can we point users to call `tracking.zoom_cancel` directly instead?

Reply:
That is what I'm doing in the Talon file. `deprecate_action` have no argument for replacement. I was kinda leaning towards maybe adding that.

Reply:
Optional replacement argument added